### PR TITLE
Fixes ash walkers being half invisible

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Zombehz
rscadd: Adds 'chicken' nuggets\* in 4 shapes, including regular, star, corgi, and lizard. it. They are made with one cutlet of meat.
/:cl:

### CODE REVIEW

**Comment by KazeEspada** [comment]
You could of given them unique sprites

**Comment by KorPhaeron** [comment]
I don't have any. Tribal lizard adventures were requested anyway. They also make sense coming from eggs

**Comment by KazeEspada** [comment]
Take the lizard sprite, add some spikes along the skin, turn the sprite grey. Remove mutcolors, enforce default color to grey. That way you have a race of spiky lizard people. Even better if you can force their features to be specific.

**Comment by Iamgoofball** [comment]
or instead of using lizards, make them rockmen

**Comment by KorPhaeron** [comment]
@Iamgoofball 

Rockmen tribes are planned but not this PR

**Comment by KazeEspada** [comment]
We have rockmen, they are already coded.

**Comment by KorPhaeron** [comment]
I know, but we don't have rockmen spawners yet

**Comment by KazeEspada** [comment]
They are called runes. JK.

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 1]
```diff
@@ -217,3 +217,31 @@
...
+	mob_name = "Penitent Exile"
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "sleeper"
+	roundstart = FALSE
+	death = FALSE
+	mob_species = /datum/species/shadow
+	flavour_text = {"You are cursed! Many years ago you risked it all to reach the Wish Granter, siezing it's power for yourself and leaving your friends for dead.. Though your wish came true, it did so at a price, and you've been doomed to wander these wastes ever since. You seek only to atone now, to somehow redeem yourself, and finally be released. You've seen ships landing in the distance. Perhaps now is the time to make things right?"}
+
+/obj/effect/mob_spawn/human/seed_vault/special(mob/living/new_spawn)
+	new_spawn.real_name = "[new_spawn.real_name] ([rand(0,999)]"
```
Comment (KazeEspada) [inline code/modules/ruins/lavaland_ruin_code.dm]: missing )

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 2]
```diff
@@ -214,3 +214,34 @@
...
+//Wishgranter Exile
+
+/obj/effect/mob_spawn/human/exile
+	name = "exile sleeper"
+	mob_name = "Penitent Exile"
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "sleeper"
+	roundstart = FALSE
+	death = FALSE
+	mob_species = /datum/species/shadow
```
Comment (KazeEspada) [inline code/modules/ruins/lavaland_ruin_code.dm]: Missing )

**Comment by Cheridan** [comment]
Oh I thought that this was a feature. Like shit's so bad down there they're all falling apart and missing limbs.

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 3]
```diff
@@ -214,3 +214,34 @@
...
+	flavour_text = {"You are cursed! Many years ago you risked it all to reach the Wish Granter, siezing it's power for yourself and leaving your friends for dead.. Though your wish came true, it did so at a price, and you've been doomed to wander these wastes ever since. You seek only to atone now, to somehow redeem yourself, and finally be released. You've seen ships landing in the distance. Perhaps now is the time to make things right?"}
+
+/obj/effect/mob_spawn/human/seed_vault/special(mob/living/new_spawn)
+	new_spawn.real_name = "[new_spawn.real_name] ([rand(0,999)]"
```
Comment (KazeEspada) [inline code/modules/ruins/lavaland_ruin_code.dm]: Missing )

**Comment by KorPhaeron** [comment]
Thanks @KazeEspada

**Pull request merged**